public/js/common/react_modal.js

```diff
@@ -13,93 +15,80 @@ export var Modal = React.createClass({
         return {
             courseId: '',
             course: [],
-            sessions: []
+            sessions: [],
+            modalIsOpen: false,
+            courseTitle : '',
+            formatCourse : '',
```

Comment:
I'm using formatCourse as a state so that I only format a course once and then pass that as a parameter for the getCourseTitle function (which previously used to call the format course function)